src/app/core.py

```diff
@@ -316,7 +317,7 @@ def license_validate_helper(request):
             fs = FileSystemStorage(location=settings.MEDIA_ROOT +"/"+ folder,
                 base_url=urljoin(settings.MEDIA_URL, folder+'/')
                 )
-            filename = fs.save(myfile.name, myfile)
+            filename = fs.save(unicodedata.normalize('NFKD', myfile.name).encode('ASCII', 'ignore').decode('ASCII'), myfile)
```

Comment:
@BanulaKumarage Can you please explain the changes. There seems to be a lot of encode/decode happening. Since this is the main flow, don't want to merge until I understand what exactly it does.

Also, can you create a util to do this since this would be required at other places as well.

Also, having some UT for this would be good.

Reply:
`unicodedata.normalize` method, converts any special character into their equivalent base characters or combinations of base characters. (ex: converting _é_ to _e_ ) Then the text is converted to ASCII characters. But if there is any character cannot convert it is replaced with a empty string( done by `ignore`) 
Finally, the resulting ASCII-encoded string is decoded back into a regular Python string, which can then be used as the filename. ( converting `b'mariadblatest.spdx'` to `mariadblatest.spdx` )